internal/telemetry/event.go

```diff
@@ -61,6 +61,18 @@ func withPrompt(p, k string) eventOpt {
 	}
 }
 
+func withChoice(c string) eventOpt {
+	return func(event Event) {
+		event.Properties["choice"] = sanitizeSelectOption(c)
+	}
+}
+
+func sanitizeSelectOption(v string) string {
+	parenthesesRegex := regexp.MustCompile(`^.*\(([^\(\)]*)\)$`)
```

Comment:
Just out of interest, do you have some examples of select options that are covered by this?

My understanding of this regex pattern is that:

- `abc(xyz) `will be sanitized to `abc()`
- `abc((xyz))` will remain unchanged

Reply:
`atlas config init` has OrgID and ProjectID that will follow the pattern `Org Name (OrgID)`